automated install/basic-install.sh

```diff
@@ -2291,7 +2437,7 @@ copy_to_install_log() {
     sed 's/\[[0-9;]\{1,5\}m//g' < /proc/$$/fd/3 > "${installLogLoc}"
 }
 
-main() {
+install_main() {
```

Comment:
Leave this function name as `main()`. All the scripts have (or will have) the main entrypoint to the installation as `main()` by convention.

Reply:
I've changed it back to its original name.